docs/integrations/web-servers/opentelemetry/apache-opentelemetry.md

```diff
@@ -0,0 +1,297 @@
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Apache-OpenTelemetry/Apache-Schematics.png')} alt="Schematics" />
+
+The Sumo Logic App for Apache assumes:
+
+-   The [NCSA extended/combined log file format ](http://httpd.apache.org/docs/current/mod/mod_log_config.html)has been configured for Apache access logs and the default error log format for Apache Access logs and Apache Error logs. For a list of metrics that are collected and used by the app, see [Apache Metrics](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/apachereceiver/documentation.md).
+
+## Fields Creation in Sumo Logic for Apache
+
+Following are the [Fields](https://help.sumologic.com/docs/manage/fields/) which will be created as part of Apache App install if not already present.
+
+- **`webengine.cluster.name`** - User configured.Enter a name to uniquely identify your Apache web server cluster. This web server cluster name will be shown in the Sumo Logic dashboards.
+- **`webengine.system`** - Has fixed value of apache
+- **`sumo.datasource`** - Has fixed value of apache
+- **`webengine..node.name`** - Has the value of host name of the machine which is being monitored
+
+## Prerequisites
+
+The receiver used, gets stats from an Apache Web Server instance using the server-status?auto endpoint.
+
+This receiver supports Apache Web Server version 2.4+
+
+In order to receive server statistics, you must configure the server's httpd.conf file to [enable status support.](https://httpd.apache.org/docs/2.4/mod/mod_status.html)
+
+Configure Apache to log to a local files. Before you can configure Sumo Logic to ingest logs, you must configure the logging of access logs and error logs via the instructions described in their [documentation](https://httpd.apache.org/docs/2.4/logs.html).
+
+To configure the Apache log files, locate your local `httpd.conf` configuration file in the Apache directory. After determining the location of the conf file, modify the httpd.conf configuration file logging parameters if required.
+
+For access logs, the following directive is to be noted:
+-   CustomLog: access log file path and format (standard common and combined)
+
+For error logs, following directives are to be noted:
+-   ErrorLog: error log file path
+-   LogLevel: to control the number of messages logged to the `error_log`
+
+## Collecting Logs, Metrics & Installing App for Apache
+
+Here are the steps for Collecting Logs, metric and installing the app:
+
+### Step 1: Set up Collector
+
+:::note
+If you want to use an existing OpenTelemetry Collector, you can skip this step by selecting the "Use an existing Collector" option.
+:::
+
+If you want to use an existing OpenTelemetry Collector then this step can be skipped by selecting the option of using an existing Collector.
+
+If you want to create a new Collector please select **Add a new Collector** option.
```

Comment:
To create a new Collector:

1. Select the Add a new Collector option.

2. Select the platform for which you want to install the Sumo OpenTelemetry Collector.